src/main/java/edu/kis/powp/jobs2d/events/SelectTransformCommandOptionListener.java

```diff
@@ -22,6 +22,6 @@ public SelectTransformCommandOptionListener(DriverManager driverManager, Complex
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        driverManager.transformCurrentDriver(transformerCommands, name);
+        driverManager.transformCurrentDriver((ComplexTransformerCommand) transformerCommands, name);
```

Comment:
Now, there is no transformCurrentDriver method in DriverManager! (?)
BTW: Nevertheless, casting should not be necessary.